sdk/identity/azure_identity/tools/deployed_live_test/src/main.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use axum::{
+    extract::Query,
+    http::StatusCode,
+    response::{IntoResponse, Response},
+    routing::get,
+    Router,
+};
+use azure_core::credentials::TokenCredential;
+use azure_identity::{ManagedIdentityCredential, ManagedIdentityCredentialOptions, UserAssignedId};
+use azure_storage_blob::BlobServiceClient;
+use serde::Deserialize;
+use std::sync::Arc;
+
+#[derive(Deserialize)]
+struct Params {
+    #[serde(rename = "client-id")]
+    client_id: Option<String>,
+    #[serde(rename = "object-id")]
+    object_id: Option<String>,
+    #[serde(rename = "resource-id")]
+    resource_id: Option<String>,
+    #[serde(rename = "storage-name")]
+    storage_name: String,
```

Comment:
Tip: you can save yourself some typing by just putting `#[serde(rename_all = "kebab-case")]` on the struct instead of on every member. We do this in generation because it's easier but hand-written code would look cleaner like this.